github.js

```diff
@@ -234,12 +258,12 @@ GithubLocation.prototype = {
         // redirect
         if (res.statusCode == 301)
           resolve({ redirect: self.name + ':' + res.headers.location.split('/').splice(3).join('/') });
-        
+
         if (res.statusCode == 401)
           reject('Invalid authentication details. Run %jspm endpoint config ' + self.name + '% to reconfigure.');
 
         // it might be a private repo, so wait for the lookup to fail as well
-        if (res.statusCode == 404 || res.statusCode == 200)
+        if (res.statusCode == 404 || res.statusCode == 200 || res.statusCode === 302)
```

Comment:
Btw, I had to add this statusCode here, because GitHub Enterprise will redirect to a login page with 302 error code (a temporary redirect).